Resources/doc/annotations-reference.rst

```diff
@@ -74,7 +74,6 @@ View
 
     /**
      * @View(
-     *  templateVar="",
      *  statusCode=null,
      *  serializerGroups={},
      *  populateDefaultVars=true,
```

Comment:
this can be removed

Reply:
it can also be removed from the view annotation

Reply:
done in #2089